qb/config.libs.sh

```diff
@@ -23,6 +23,7 @@ SOCKETLIB=-lc
 SOCKETHEADER=
 
 if [ "$OS" = 'BSD' ]; then
+   [ -d /usr/local/include ] && add_dirs INCLUDE /usr/local/include
```

Comment:
While this works, you are also missing adding `/usr/local/lib` for any libraries, otherwise while it finds the headers, it wont be able to find the library needed.